libs/shared/node/src/interfaces/nx-json.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license
+ * Copyright 2020 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface NxJson {
+  npmScope: string;
+  implicitDependencies: {
+    [key: string]: string;
```

Comment:
You can go into the implicit dependencies as well and specify parts of it

```suggestion
    [key: string]: { [key: string]: string } | string;
```


like: 

```json
{
"implicitDependencies": {
    "package.json": {
      "dependencies" : "*"
    },
    "nx.json": "*",
  }
}
```